File: MatrixScrabble.Server/Models/Context/Game.cs
namespace MatrixScrabble.Server.Models.Сontext;

public partial class Game
{
	public Guid Id { get; set; }
	public required string Word { get; set; }
	public required string Language { get; set; }
	public bool IsCompleted { get; set; }
	public DateTime DateCreated { get; set; }
	public string? Board { get; set; }
}

File: matrixscrabble.client/src/redux/models/Game.ts
import { Language } from "../../constants";
import { GameBoardModel } from "./GameBoardModel";
import { GameDetails } from "./GameDetails";
import { NewGameBoard } from "./NewGameBoard";

export interface Game {
  id: string;
  word: string;
  isCompleted?: boolean;
  language: Language;
  board?: GameBoardModel;
  gameBoard: NewGameBoard;
  details?: GameDetails;
}
